api/lib/controllers/institutions/actions.js

```diff
@@ -129,6 +147,22 @@ exports.updateInstitution = async (ctx) => {
     throw new Error(e);
   }
 
+  let correspondents = await institution.getContacts();
+  correspondents = correspondents.map((e) => e.email);
+
+  if (correspondents.length > 0) {
+    if (!wasValidated && body.validated === true) {
```

Comment:
```suggestion
  if (!wasValidated && body.validated === true) {
    let correspondents = await institution.getContacts();
    correspondents = correspondents?.map?.((e) => e.email);

    if (correspondents.length > 0) {
```

No need to fetch contacts if we don't send the mail